### PR TITLE
added a checkbox to show/hide betas

### PR DESCRIPTION
**Pull request by dktapps**
this is not currently saved after a restart because there currently doesn't seem to be any way to save user preferences.

image
image

### CODE REVIEW

File: MCLauncher/MainWindow.xaml [hunk 1]
```diff
@@ -59,8 +59,8 @@
             </Grid>
         </DataTemplate>
     </Window.Resources>
-    <Grid>
-        <ListBox Name="VersionList" Margin="10,10,10,10"  d:DataContext="{d:DesignData /SampleData/Versions.xaml}" ItemsSource="{Binding}" HorizontalContentAlignment="Stretch">
+    <Grid Margin="0,0,0,29">
```
Comment (MCMrARM) [inline MCLauncher/MainWindow.xaml]: Would be nice if this Margin could be removed from here and replaced with nice numbers in the ListBox instead.

File: MCLauncher/MCLauncher.csproj [hunk 2]
```diff
@@ -108,6 +108,7 @@
       <Generator>MSBuild:Compile</Generator>
       <SubType>Designer</SubType>
     </ApplicationDefinition>
+    <Compile Include="Preferences.cs" />
```
Comment (MCMrARM) [inline MCLauncher/MCLauncher.csproj]: Missing?

File: MCLauncher/MainWindow.xaml.cs [hunk 3]
```diff
@@ -33,8 +37,22 @@ public partial class MainWindow : Window, ICommonVersionCommands {
 
         public MainWindow() {
             InitializeComponent();
+            ShowBetasCheckbox.DataContext = this;
+
+            if (File.Exists(PREFS_PATH))
```
Comment (MCMrARM) [inline MCLauncher/MainWindow.xaml.cs]: Project uses different codestyle.

File: MCLauncher/MainWindow.xaml (hunk 1 above)
Comment (dktapps) [reply MCLauncher/MainWindow.xaml]: I kept a 10,10,10,10 on the grid for consistency's sake. The rest was to make space for the bottom grid row, which I've cleaned up.

**Pull request merged**